packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -8,8 +8,8 @@ import 'dart:io';
 import 'dart:ui' as ui;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
```

Comment:
Don't import services.dart, import the individual libraries you need from services, since this library is itself part of services